addons/sale/models/sale_order.py

```diff
@@ -520,11 +520,13 @@ def create(self, vals):
         return result
 
     def _compute_field_value(self, field):
+        if field.name == 'invoice_status':
```

Comment:
I know I missed that in 13.0 (sorry for that, I will take care of it) but can we add this starting from here?
```suggestion
        if field.name == 'invoice_status' and not self.env.context.get('mail_activity_automation_skip'):
```

Thanks 🙂

Reply:
Fixing 14.0 has been done in #79467

Reply:
@robinengels I made the necessary changes in order to proceed 🙂